Automattic/pocket-casts-android
# Add free trial info to Plus settings screen

# Description

Project https://github.com/Automattic/pocket-casts-android/issues/190

Adding trial info to the Plus settings screen. 

I also switched the order of the "$ per year / $ per month" to be "$ per month / $ per year" to be consistent with iOS and because I thought it made sense to put the smaller number first. I don't feel strongly here though, so let me know if you disagree.

| Before | After No Trial | After with trial (top) | After with trial (bottom) |
| --- | --- | --- | --- |
| ![image](https://user-images.githubusercontent.com/4656348/183228836-014e22ae-64f6-420f-a1d2-07ac20be2f0f.png) | ![image](https://user-images.githubusercontent.com/4656348/183229307-69e4f901-9f73-42cc-8e5f-1235fbadea7b.png) | ![image](https://user-images.githubusercontent.com/4656348/183229212-a3a645fb-9055-430f-b2b7-f8a5740c3a4e.png) | ![image](https://user-images.githubusercontent.com/4656348/183229218-818505e6-6b46-4e02-83f4-6e603a2ff4b4.png) |





### Test Steps

1. Log out of the app
2. Go to the profile tab
3. Tap the cog on the top-right of the screen
4. Tap "Pocket Casts Plus"
5. Observe that the button says "Start Free Trial" and the trial info is below the button
6. Scroll to the bottom of the screen and observe the same
7. Adjust the app to not have a free trial available (I like to just replace the string assigned [here](https://github.com/Automattic/pocket-casts-android/blob/91d4a5f14a614ebb014377a1bba18bf457fdb478/modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/subscription/SubscriptionManager.kt#L19) to be the string for either the monthly or yearly product)
8. Observe that the buttons now say "Upgrade to Plus" and the monthly and yearly pricing info is given below the buttons.

# Checklist
